lua/neogit/popups/ignore/actions.lua

```diff
@@ -0,0 +1,58 @@
+local M = {}
+
+---@param path string
+---@param rules string[]
+local function add_rules(path, rules)
+  local async = require("plenary.async")
+  local notification = require("neogit.lib.notification")
+
+  local err, fd = async.uv.fs_open(path, "a+", 438)
+  if err then
+    return notification.error(string.format("Failed to read ignore file %q\n\n%s", path, err))
+  end
+
+  async.uv.fs_write(fd, table.concat(rules, "\n") .. "\n")
+
+  local err = async.uv.fs_close(fd)
+  if err then
+    return notification.error(string.format("Failed to close file %q", path))
+  end
+
+  -- notification.info(string.format("Added %d rules to %q", #rules, path))
+end
+
+local operation = require("neogit.operations")
+M.shared = operation("ignore_shared", function(popup)
```

Comment:
I have a feeling `git.repo.git_path()` could be of use here: https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/lib/git/repository.lua#L15-L17

Reply:
Fixed :smile: